src/core/lib/iomgr/resource_quota.cc

```diff
@@ -665,7 +665,7 @@ void grpc_resource_quota_unref_internal(grpc_resource_quota* resource_quota) {
     GPR_ASSERT(resource_quota->num_threads_allocated == 0);
     GRPC_COMBINER_UNREF(resource_quota->combiner, "resource_quota");
     gpr_free(resource_quota->name);
-    gpr_mu_destroy(&resource_quota->thread_count_mu);
+    // gpr_mu_destroy(&resource_quota->thread_count_mu);
```

Comment:
did you commented these lines to see the failures?

Reply:
Yes, but the leaks haven't been caught this time, there must be something wrong with this approach.

The expected tests' log should containing the leaks like [this](https://source.cloud.google.com/results/invocations/68bbd049-f56b-4bd5-adbf-ec6283adf379/log) which was reported by previous version's test.

Reply:
Reverted.